### PR TITLE
solution

### PR DESCRIPTION
**Pull request by the-tique**
[DEMO LINK](https://the-tique.github.io/react_movies-list-js/)

### CODE REVIEW

File: src/components/MovieCard/MovieCard.jsx [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={movie.imgUrl}
```
Comment (loralevitska) [inline src/components/MovieCard/MovieCard.jsx:10]: Destructure `movie`

File: src/components/MovieList/MovieList.jsx [hunk 2]
```diff
@@ -1,5 +1,12 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <>
```
Comment (loralevitska) [inline src/components/MovieList/MovieList.jsx:5]: Remove redundant [`React.Fragment`](https://www.memberstack.com/blog/react-fragment)

File: src/components/MovieList/MovieList.jsx [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+    <div className="movies">
+      {movies.map(movie => (
+        <MovieCard key={movie.imdbId} movie={movie} />
+      ))}
+    </div>
```
Comment (loralevitska) [inline src/components/MovieList/MovieList.jsx:10]: Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost done) Let's improve

File: src/components/MovieList/MovieList.jsx (hunk 3 above)
Comment (the-tique) [reply src/components/MovieList/MovieList.jsx]: not a single word about that in checklist tho

**Pull request closed without merging**